base/applications/sndrec32/CMakeLists.txt

```diff
@@ -15,6 +15,6 @@ add_rc_deps(rsrc.rc ${sndrec32_rc_deps})
 add_executable(sndrec32 ${SOURCE} rsrc.rc)
 target_link_libraries(sndrec32 cpprt)
 set_module_type(sndrec32 win32gui UNICODE)
-add_importlibs(sndrec32 winmm user32 msacm32 comctl32 comdlg32 gdi32 shell32 msvcrt kernel32)
+add_importlibs(sndrec32 winmm user32 msacm32 comctl32 comdlg32 gdi32 shell32 msvcrt kernel32 uxtheme)
```

Comment:
uxtheme has to be loaded dynamically.

Reply:
1.) It would be more correct, As XPSP3 does not have that static import to uxtheme in sndrec32. I double-checked with MSDepends for XPSP3. 2k3sp2 with almost certainty also doesn't have that.
2.) It will continue to allow running sndrec in Win2000, XPSP3/2k3sp2-nlite-slipstreamed. And maybe also on XPSP2embedded, where it might be possible to exclude uxtheme as well.

Reply:
We don't target `Win2000, XPSP3/2k3sp2-nlite-slipstreamed. And maybe also on XPSP2embedded`

Reply:
But we do target 2k3sp2. And do intend to do the things like 2k3sp2 does them.